### PR TITLE
docs: Update links and name

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
Clean names and update links to `hexo-theme-doc`.

Add dependency status badge.

### CODE REVIEW

**Comment by bhaskarmelkani** [comment]
👍

File: .zappr.yaml [hunk 1]
```diff
@@ -14,5 +12,4 @@ commit:
       # CONTRIBUTING.md#-git-commit-guidelines
       - "^(feat|fix|docs|style|refactor|perf|test|chore)(|\\([a-zA-Z0-9-._]+\\)):.{3,}"
 
-X-Zalando-Team: stargate
 X-Zalando-Type: code
```
Comment (rbarilani) [inline .zappr.yaml]: @bhaskarmelkani remove also this line

**Comment by rbarilani** [comment]
👍

File: .zappr.yaml (hunk 1 above)
Comment (bhaskarmelkani) [reply .zappr.yaml]: done

**Comment by bhaskarmelkani** [comment]
👍

**Pull request merged**